seqr/views/utils/file_utils_tests.py

```diff
@@ -96,15 +106,50 @@ def test_temp_file_upload(self):
             'parsedData': PARSED_DATA,
             'uploadedFileId': mock.ANY,
         })
+        # Remove temp file
+        load_uploaded_file(response_json['uploadedFileId'])
+
+        # Test uploading with csv data
+        response = self.client.post('{}?parsedData=true'.format(url),
+                        {'f': SimpleUploadedFile("test_data.csv", CSV_DATA)})
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertDictEqual(response_json, {
+            'parsedData': PARSED_DATA,
+            'uploadedFileId': mock.ANY,
+        })
+        # Remove temp file
+        load_uploaded_file(response_json['uploadedFileId'])
+
+        # Test uploading with xlsx data
+        wb = xl.Workbook()
+        ws = wb[wb.sheetnames[0]]
+        ws['A1'], ws['B1'], ws['C1'] = XLSX_PARSED_DATA[0]
+        ws['A2'], ws['B2'], ws['C2'] = XLSX_PARSED_DATA[1]
+        ws['A3'], ws['B3'], ws['C3'] = XLSX_PARSED_DATA[2]
+        with NamedTemporaryFile() as tmp:
```

Comment:
setup code must be done in the test setup, not in the middle of the test

Reply:
Moved to setUp()